snapcraft/cli/_errors.py

```diff
@@ -110,6 +108,57 @@ def exception_handler(exception_type, exception, exception_traceback, *,
     sys.exit(exit_code)
 
 
+def _is_send_to_sentry() -> bool:
+    # If ALWAYS has already been selected from before do not even bother to
+    # prompt again.
+    config_errors = None
+    try:
+        cli_config = _CLIConfig()
+        always_value = cli_config.get_option('Sentry', 'always_send')
+        if always_value == 'true':
+            return True
+    except errors.SnapcraftInvalidCLIConfigError as config_error:
+        raise config_error
+        echo.warning('There was an issue while trying to retrieve '
```

Comment:
This probably isn't what you want. Do we want to re-raise, or print a warning?

Reply:
ooh, I was playing with this, yes, just the warning, we cannot re raise as we are in the exception already